src/sass/variables.scss

```diff
@@ -1,9 +1,19 @@
 
 // Global Colors
 $dh-color-white: white;
+$dh-color-gray80: #cccccc;
 
+$dh-link_color-default: #005ea2;
+$dh-link_color-hover: #1a4480;
+$dh-link_color-visited: purple;
+$dh-link_text-decoration: underline;
 
 // Footer variables
 $dh-footer-color-primary: $theme-color-base-lightest;
 $dh-footer-color-secondary: #003E7E;
 
+// About page variables
+$dh-about_text-width: 110ex;
+$dh-about_font-size: 16px;
```

Comment:
These should probably be rems.

Reply:
You can use the built in USWDS stuff to make everything scale correctly, here's an example of my header scss. USWDS uses ambiguous "units". They seem to allow .5 increments and whole numbers.
#nav_top{
    background-color: $dh-color-dot-banner-background; 
    width: 100%;
    height: 5rem;
    height: units(10);
    padding-top: units(2.5);
    padding-bottom: units(2.5);
}